add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -567,11 +568,11 @@ var LoopRoomsInternal = {
           eventEmitter.emit("delete", room);
           eventEmitter.emit("delete:" + room.roomToken, room);
         } else {
+          yield this.addOrUpdateRoom(room, !!orig);
+
```

Comment:
We need to manually audit the (few, I hope) callers of this to make sure this change won't cause problems for them.